combat-trainer.lic

```diff
@@ -384,6 +388,11 @@ class LootProcess
     return if Time.now - @loot_timer < @loot_delay
 
     game_state.mob_died = true
+    if (Time.now - @last_rites_timer > 600) && @last_rites
+      fput "pray #{DRRoom.dead_npcs.first}"
```

Comment:
need to grab a message if it's already skinned, if it's cursed, undead (maybe those are the same), or done too soon